packages/@aws-cdk/aws-ecs/test/test.app-mesh-proxy-configuration.ts

```diff
@@ -9,18 +9,21 @@ export = {
     const stack = new cdk.Stack();
 
     // WHEN
-    const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'Ec2TaskDef', { proxyConfiguration: ecs.ProxyConfigurations.appMeshProxyConfiguration({
-      containerName: "envoy",
-      properties: {
-        ignoredUID: 1337,
-        ignoredGID: 1338,
-        appPorts: [80, 81],
-        proxyIngressPort: 80,
-        proxyEgressPort: 81,
-        egressIgnoredPorts: [8081],
-        egressIgnoredIPs: ["169.254.170.2", "169.254.169.254"],
-      }
-    })});
+    const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'Ec2TaskDef', {
+      networkMode: ecs.NetworkMode.AWS_VPC,
```

Comment:
Should the `networkMode` default be changed from `BRIDGE` with `proxyConfiguration`?

https://github.com/aws/aws-cdk/blob/40d167475f0547b45e674cdc252ad7d24ba03f2f/packages/%40aws-cdk/aws-ecs/lib/base/task-definition.ts#L256-L257

Reply:
No, I don't think it is necessary and will be more confusing if users specify the networkMode.

Reply:
I would actually rely on the user to provide the right NetworkMode (AWS_VPC) if they're using proxy configuration.